tests/test_graph_api.py

```diff
@@ -684,6 +685,7 @@ def test_get_unicode_url():
     mock_request.return_value.status_code = 200
 
     response = graph.get('https://www.facebook.com/christophernewportuniversity‎')
+    del response['headers']
```

Comment:
Not at all! You're testing that facepy extracts headers now – it doesn't matter which ones. Thanks!